.github/workflows/nettrine-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['7.4']
+
+      fail-fast: false
+
+    services:
+      mysql:
+        image: mariadb:latest
+        env:
+          MYSQL_USER: user
+          MYSQL_PASSWORD: password
+          MYSQL_DATABASE: nettrine
+          MYSQL_ROOT_PASSWORD: password
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+        ports:
+          - 3306:3306
+
+    name: PHP ${{ matrix.php }} Nettrine tests
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v1
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: json, mbstring, xml, tokenizer
+          coverage: none
+
+      - run: mkdir -p nettrine/tests/tmp
+      - run: cat $(php -r 'echo php_ini_loaded_file() . " " . str_replace("\n", " ", str_replace(",", "", php_ini_scanned_files()));') > nettrine/tests/tmp/php.ino
+      - run: cd nettrine && composer install --no-interaction --prefer-source
+      - run: cp nettrine/app/config/local.github-action.neon nettrine/app/config/local.neon
```

Comment:
Misto takoveho souboru v repu, pouzij prosim neonizer (https://github.com/contributte/neonizer/blob/master/.docs/README.md#set-variables) a nasetuj to do local.neon.

Reply:
hotovo